controller/customdomain_utils.go

```diff
@@ -228,3 +233,39 @@ func randSeq(n int) string {
 	}
 	return string(b)
 }
+
+// GetPlatformType returns the cloud platform type for the cluster
+func GetPlatformType(kclient client.Client) (*configv1.PlatformType, error) {
```

Comment:
It looks like these were just copied from https://github.com/openshift/cloud-ingress-operator/blob/master/pkg/utils/infrastructure.go
Why don't you just import "github.com/cloud-ingress-operator/pkg/utils" instead?

Reply:
Hey Dustin. I was doing this for some operators but at some I believe Christoph or Taylor, I don't remember which one told me is not that good of a practice importing one operator into another (I might dreaming on this but I vaguely remember having this conversation with one of the leads). This is why Instead of importing it I just got the full code. But this import was my first instinct also, so I do not mind making this change if it is a must for you, Let me know.

Reply:
Fair enough, the code is small enough to not worry about the replication at this point.